imports/startup/client/toastr.coffee

```diff
@@ -0,0 +1,22 @@
+toastr.options =
+  'closeButton': true
+  'debug': false
+  'newestOnTop': false
+  'progressBar': false
+  'positionClass': 'toast-bottom-full-width'
+  'preventDuplicates': true
+  'onclick': null
+  'showDuration': '500'
+  'hideDuration': '1000'
+  'timeOut': 0
+  'extendedTimeOut': 0
+  'showEasing': 'swing'
+  'hideEasing': 'linear'
+  'showMethod': 'fadeIn'
+  'hideMethod': 'fadeOut'
+  'tapToDismiss': false
+
+Meteor.toastr = (err) ->
```

Comment:
Could you use toastr.error directly rather than a wrapper function? I would like to keep the loading indicator code independent of the error message code.